### PR TITLE
ASP.NET 5 Detailed

### PR DESCRIPTION
**Pull request by louislewis2**
Added more information and extra screenshots to cover popular questions.

### CODE REVIEW

File: Using-TypeScript-With-ASP.NET-5.md [hunk 1]
```diff
@@ -9,7 +9,11 @@ We start by creating a new empty ASP.NET v5 project in Visual Studio 2015, of yo
  ![Create new Empty ASP.NET Project](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/aspnet-screenshots/new-project.png)
  
 Next add a `scripts` folder to the root of our project.
-This is where we'll add the TypeScript files and the [tsconfig.json](tsconfig.json.md) file to set our compiler options.
+This is where we'll add the TypeScript files and the [tsconfig.json](tsconfig.json.md) file to set our compiler options. Please note that the names and locations of
```
Comment (DanielRosenwasser) [inline Using-TypeScript-With-ASP.NET-5.md]: Please break sentences onto different lines

File: Using-TypeScript-With-ASP.NET-5.md [hunk 2]
```diff
@@ -9,7 +9,11 @@ We start by creating a new empty ASP.NET v5 project in Visual Studio 2015, of yo
  ![Create new Empty ASP.NET Project](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/aspnet-screenshots/new-project.png)
  
 Next add a `scripts` folder to the root of our project.
-This is where we'll add the TypeScript files and the [tsconfig.json](tsconfig.json.md) file to set our compiler options.
+This is where we'll add the TypeScript files and the [tsconfig.json](tsconfig.json.md) file to set our compiler options. Please note that the names and locations of 
+the folders are pertinent to get the solution working correctly. To add a tsconfig.json file, simply right click on the 'scripts' folder, navigate to 'Add', then 'New Item'.
+Under Client-side, you can find it, as can be seen below.
+
+![Project layout](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/aspnet-screenshots/add-tsconfig.png)
```
Comment (paulvanbrenk) [inline Using-TypeScript-With-ASP.NET-5.md]: This is the alt-tag used by the image, can you make this descriptive?

File: Using-TypeScript-With-ASP.NET-5.md [hunk 3]
```diff
@@ -19,11 +23,34 @@ Finally we have to add the following option to the `"compilerOptions"` node in t
...
 
 ![Files after build](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/aspnet-screenshots/postbuild.png)
 
+##Project vs Virtual Project
+When adding a tsconfig.json file, it is vital to understand that this creates a virtual typescript project within the folder
+where the tsconfig.json is located. TypeScript files that are considered part of this virtual project, will not be compiled when 
+saving changes. TypeScript files that are outside of the folder containing the tsconfig.json are considered not to be part of the virtual project.
+In the image below, the virtual project can be visualized, and is the that which is contained within the blue rectangle.
+
+![Compile on Save](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/aspnet-screenshots/virtual-project.png)
```
Comment (paulvanbrenk) [inline Using-TypeScript-With-ASP.NET-5.md]: This is used as the 'alt-tag' for the image, please make it descriptive.

**Comment by DanielRosenwasser** [comment]
Thanks @louislewis2

**Comment by louislewis2** [comment]
@DanielRosenwasser  Pleasure, and glad I could help.
PS: Sorry about the "Developer" English :)

**Comment by DanielRosenwasser** [comment]
@louislewis2 don't worry, I'm pretty fluent at Developer English :smile:

**Pull request merged**